### PR TITLE
Adds kubectl plugin list to export

### PR DESCRIPTION
**Pull request by tony-landreth**
Issue: PGO-598

### CODE REVIEW

**Review by benjaminjb — COMMENTED** [review]
LGTM, but I think we want to add documentation to the Help message re: what we're doing here with an exec.

File: docs/content/reference/pgo_support_export.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ kubectl pgo support export daisy --monitoring-namespace another-namespace --outp
 | PGO CLI Support Export Tool
 | The support export tool will collect information that is
 | commonly necessary for troubleshooting a PostgresCluster.
-| Note: No data or k8s secrets are collected.
+| Note: No data or k8s secrets are collected, however,
+| kubectl is used to list plugins on the user's machine.
```
Comment (tony-landreth) [inline docs/content/reference/pgo_support_export.md:72]: @benjaminjb Here's an attempt at documenting the exec.

File: internal/cmd/export.go [hunk 2]
```diff
@@ -231,7 +232,8 @@ kubectl pgo support export daisy --monitoring-namespace another-namespace --outp
 | PGO CLI Support Export Tool
 | The support export tool will collect information that is
 | commonly necessary for troubleshooting a PostgresCluster.
-| Note: No data or k8s secrets are collected.
+| Note: No data or k8s secrets are collected, however,
+| kubectl is used to list plugins on the user's machine.
```
Comment (benjaminjb) [inline internal/cmd/export.go:236]:
```suggestion
| Note: No data or k8s secrets are collected.
| However, kubectl is used to list kubectl plugins on the user's machine.
```
?
I just like the period after `No data or k8s secrets are collected.` It's a definitive statement.
Kubeectl to list kubectl plugins is redundant, but I want to be redundant here.

**Review by benjaminjb — APPROVED** [review]
LGTM, not a blocker, just a tweak to the language.

File: internal/cmd/export.go (hunk 2 above)
Comment (tony-landreth) [reply internal/cmd/export.go:236]: I think you're right. I was internally debating it.

**Review by benjaminjb — APPROVED** [review]

**Pull request merged**